highfidelity/hifi
# Move cellscience to production s3 bucket

This PR puts all assets and scripts into the 'production' bucket on s3 and updates references.  The 'production' bucket will actually cache stuff, vs. the 'content' bucket which sends the content new every time.  Should help with load times.  AC scripts are running off of 'production' as well.

To test:
1. Go to cellscience.
2. You should hear music and see entities.
3. Click each of the buttons
